### ISSUE TITLE
[FEATURE] $ARGS.named for --arg

### ISSUE THREAD

**Issue by tkapias** [01mf02/jaq#53]
Could you finalize the implementation of ARG from [issues/11](https://github.com/01mf02/jaq/issues/11)?

I think that implementing only --arg and not --argjson (as a way to import floats) was a good choice, but you left $ARGS.named behind.

$ARGS.named is a really nice feature to directly transfer a collection of shell variables.

``` shell
status=1
city="Paris"
year=1984

jq -n --arg status  $_status \
      --arg city "$city" \
      --arg year  $year \
      '$ARGS.named'
```

``` json
{
  "status": "1",
  "city": "Paris",
  "year": "1984"
}
```

**Comment by pkoppstein** [comment]
You must be using an old version of jq. Using a recent version:
```
jq -n --arg a 1 --arg b 1 --argjson c 1 '$ARGS'
{
  "positional": [],
  "named": {
    "a": "1",
    "b": "1",
    "c": 1
  }
}
```

**Comment by 01mf02** [comment]
Thanks for explaining your use case, this was a great motivation for me. :)
Implemented in 505ef4e5dd0708b63800161f4dc13fd194e5140e.
